src/DotNetLightning.Core/Channel/CommitmentsModule.fs

```diff
@@ -415,3 +416,80 @@ module internal Commitments =
                               OriginChannels = originChannels1 }
                 return [ WeAcceptedCommitmentSigned(nextMsg, nextCommitments) ;]
             }
+
+module ForceCloseFundsRecovery =
+    // The lightning spec specifies that commitment txs use version 2 bitcoin transactions.
+    let TxVersionNumberOfCommitmentTxs = 2u
```

Comment:
You can specify `[<Literal>]` attribute for constants like this. Which will slightly improves performance.

Reply:
Fixed.